src/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,108 @@
+import {
+	Component,
+	Input
+} from "@angular/core";
+
+import { BreadcrumbItem } from "./breadcrumb-item.interface";
+
+const MINIMUM_OVERFLOW_THRESHOLD = 4;
+
+@Component({
+	selector: "ibm-breadcrumb",
+	template: `
+	<nav #nav class="bx--breadcrumb"
+		[ngClass]="{
+			'bx--breadcrumb--no-trailing-slash' : noTrailingSlash
+		}"
+		[attr.aria-label]="ariaLabel">
+		<ng-template [ngIf]="shouldShowContent">
+			<ng-content></ng-content>
+		</ng-template>
+		<ng-template [ngIf]="!shouldShowOverflow">
+			<ibm-breadcrumb-item
+				*ngFor="let item of items"
+				[href]="item.href">
+				{{item.content}}
+			</ibm-breadcrumb-item>
+		</ng-template>
+		<ng-template [ngIf]="shouldShowOverflow">
+			<ibm-breadcrumb-item [href]="first?.href">
+				{{first?.content}}
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				<ibm-overflow-menu>
+					<li class="bx--overflow-menu-options__option"
+						*ngFor="let item of overflowItems">
+						<a class="bx--overflow-menu-options__btn"
+							href="{{item?.href}}"
+							style="text-decoration: none;">
+							{{item?.content}}
+						</a>
+					</li>
+				</ibm-overflow-menu>
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item [href]="secondLast?.href">
+				{{secondLast?.content}}
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item [href]="last?.href">
+				{{last?.content}}
+			</ibm-breadcrumb-item>
+		</ng-template>
+	</nav>`
+})
+export class Breadcrumb {
+	_threshold: number;
```

Comment:
Should be
```suggestion
	protected _threshold: number;
```

unless there's a reason for it to be public.